_P208_Nokia_LCD_5110.ino

```diff
@@ -256,7 +262,7 @@ void setBacklight(struct EventStruct *event) {
   }
 }
 
-boolean displayText3(char deviceTemplate[][48], struct EventStruct *event ){ //48 = digits_per_line
+boolean displayText3(char deviceTemplate[][48], struct EventStruct *event ){ //14 must be digits_per_template_line

```

Comment:
Shouldn't that 48 also be replaced by the define?

Reply:
Should be nice.... but that give's a error in ESPEasy.ino (I'm using PlatformIO)
"PlatformIO/Projects/ESPEasy-mega-20210223/src/ESPEasy.ino:595:44: error: 'digits_per_template_line' was not declared in this scope"

Reply:
Adding round braces like this `deviceTemplate[][(digits_per_template_line)],` seems to work. This smells like a compiler/preprocessor bug, but there may be a usecase for it. I wouldn't know.

And a few other conventions, defines like that are usually written in all uppercase, and as the .ino sources are compiled as a single source file (ESPEasy.ino, by Arduino convention), the plugin ID is usually part of that name, something like `P208_DIGITS_PER_TEMPLATE_LINE`. The same goes for the method name, should best be something like `p208_DisplayText3(...)`.

Reply:
And adding a forward declaration to the source before the method is used also helps avoiding this error message.

Reply:
I tried all your suggestions without result.